gtest_parallel_tests.py

```diff
@@ -150,19 +150,20 @@ def assertTaskWasExecuted(self, test_id, expected, retries):
     self.times.assertRecorded(self,test_id, expected, retries)
     self.test_results.assertRecorded(self,test_id, expected, retries)
 
-  def test_run_task_basic(self):
+  def retry_failed_n_times(self, retry_failed, n_tasks, expected_exit_code):
```

Comment:
Can you make these named parameters, and below run with retry_failed=X instead of X, this makes the function calls easier to read.